src/components/HorizontalTab/index.tsx

```diff
@@ -6,15 +7,10 @@ import {colors} from '../../utils/colors';
 
 const {width} = Dimensions.get('screen');
 
-export type HorizontalTabItemProps = {
-  key: number;
-  tabElement: React.ReactNode;
-};
-
 export type HorizontalTabProps = {
   selectedTab: number;
   onSelectedTabChange: (index: number) => void;
-  tabs: HorizontalTabItemProps[];
+  tabs: React.ReactNode[];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The type of the `tabs` prop has been changed from an array of objects with a specific structure (`HorizontalTabItemProps[]`) to an array of `ReactNode`s. This change reduces the type safety of the component and makes it less clear what kind of data should be passed in the `tabs` prop. Consider reverting this change or providing a more specific type for the `tabs` prop.

```diff
- tabs: React.ReactNode[];
+ tabs: HorizontalTabItemProps[];
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->